tidb-cloud/limited-sql-features.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Limited SQL Features on TiDB Cloud
+summary: Learn about the limited SQL features on TiDB Cloud.
+---
+
+# Limited SQL features on TiDB Cloud
+
+TiDB Cloud works with almost all workloads that TiDB supports, but there are feature differences between TiDB Self-Hosted and TiDB Dedicated/Serverless. This document describes the limitations of SQL features on TiDB Cloud. We are constantly filling in the feature gaps between TiDB Self-Hosted and TiDB Dedicated/Serverless. If you require these features or capabilities in the gap, [contact us](/tidb-cloud/tidb-cloud-support.md) for a feature request.
+
+## Statements
+
+### Placement and range management
+
+| Statement | TiDB Dedicated | TiDB Serverless |
+|:-|:-|:-|
+| `ALTER PLACEMENT POLICY` | Supported | Not supported [^1] |
+| `CREATE PLACEMENT POLICY` | Supported | Not supported [^1] |
+| `DROP PLACEMENT POLICY` | Supported | Not supported [^1] |
+| `SHOW CREATE PLACEMENT POLICY` | Supported | Not supported [^1] |
+| `SHOW PLACEMENT` | Supported | Not supported [^1] |
+| `SHOW PLACEMENT FOR` | Supported | Not supported [^1] |
+| `SHOW PLACEMENT LABELS` | Supported | Not supported [^1] |
+| `SHOW TABLE REGIONS` | Supported | Not supported [^1] |
+| `SPLIT REGION` | Supported | Not supported [^1] |
+
+### Resource groups
+
+| Statement | TiDB Dedicated | TiDB Serverless |
+|:-|:-|:-|
+| `ALTER RESOURCE GROUP` | Supported | Not supported [^2] |
+| `CALIBRATE RESOURCE` | Supported | Not supported [^2] |
+| `CREATE RESOURCE GROUP` | Supported | Not supported [^2] |
+| `DROP RESOURCE GROUP` | Supported | Not supported [^2] |
+| `SET RESOURCE GROUP` | Supported | Not supported [^2] |
+| `SHOW CREATE RESOURCE GROUP` | Supported | Not supported [^2] |
+
+### Others
+
+| Statement | TiDB Dedicated | TiDB Serverless |
+|:-|:-|:-|
+| `BACKUP` | Supported | Not supported [^3] |
+| `SHOW BACKUPS` | Supported | Not supported [^3] |
+| `RESTORE` | Supported | Not supported [^3] |
+| `SHOW RESTORES` | Supported | Not supported [^3] |
+| `ADMIN RESET TELEMETRY_ID` | Supported | Telemetry is not supported on TiDB Serverless. |
+| `ADMIN SHOW TELEMETRY` | Not supported [^4] | Not supported [^4] |
```

Comment:
Is `[^4]` here correct for the TiDB Serverless column?   The previous row indicates that Telemetry is not supported on TiDB Serverless.

Reply:
Yes, the second SQL is disabled by SEM therefore is not supported on both dedicated and serverless.

`ADMIN RESET TELEMETRY_ID` is not restricted by SEM, therefor works on dedicated and we explicitly disabled it on serverless.